diff --git a/IClientController.h b/IClientController.h
index 44c17bf..a1b55dd 100644
--- a/IClientController.h
+++ b/IClientController.h
@@ -5,7 +5,7 @@ public:
 #ifdef ARGONX_UNIX
     virtual unknown_ret __Destructor2() = 0;
 #endif
-  virtual unknown_ret Init(bool, class IClientControllerSerialized*, unsigned int) = 0;
+    virtual unknown_ret Init(bool, class IClientControllerSerialized*, unsigned int) = 0;
     virtual unknown_ret Shutdown() = 0;
     virtual unknown_ret RunFrame() = 0;
     virtual unknown_ret GetAllControllersStatus(ControllersStatus_t*) = 0;
diff --git a/IClientEngine.h b/IClientEngine.h
index 4c36e32..1396372 100644
--- a/IClientEngine.h
+++ b/IClientEngine.h
@@ -49,7 +49,7 @@ public:
     virtual void *GetIClientAudio(Steam::UserHandle h, Steam::PipeHandle p) = 0;
     virtual void *GetIClientMusic(Steam::UserHandle h, Steam::PipeHandle p) = 0;
     virtual void *GetIClientUnifiedMessages(Steam::UserHandle h, Steam::PipeHandle p) = 0;
-    virtual void *GetIClientController(Steam::UserHandle h, Steam::PipeHandle p) = 0;
+    virtual void *GetIClientController(Steam::PipeHandle p) = 0;
     virtual void *GetIClientParentalSettings(Steam::UserHandle h, Steam::PipeHandle p) = 0;
     virtual void *GetIClientStreamLauncher(Steam::UserHandle h, Steam::PipeHandle p) = 0;
     virtual void *GetIClientDeviceAuth(Steam::UserHandle h, Steam::PipeHandle p) = 0;
@@ -72,12 +72,12 @@ public:
     virtual void * GetIClientCompat(Steam::UserHandle h, Steam::PipeHandle p) = 0;
     virtual unknown_ret SetClientCommandLine(int, char**) = 0;
     virtual void * GetIClientParties(Steam::UserHandle h, Steam::PipeHandle p) = 0;
-    virtual unknown_ret GetIClientNetworkingMessages(int, int) = 0;
-    virtual unknown_ret GetIClientNetworkingSockets(int, int) = 0;
-    virtual unknown_ret GetIClientNetworkingUtils(int) = 0;
-    virtual unknown_ret GetIClientNetworkingUtilsSerialized(int) = 0;
-    virtual unknown_ret GetIClientSTARInternal(int, int) = 0;
-    virtual unknown_ret GetIClientRemotePlay(int, int) = 0;
+    virtual void *GetIClientNetworkingMessages(int, int) = 0;
+    virtual void *GetIClientNetworkingSockets(int, int) = 0;
+    virtual void *GetIClientNetworkingUtils(int) = 0;
+    virtual void *GetIClientNetworkingUtilsSerialized(int) = 0;
+    virtual void *GetIClientSTARInternal(int, int) = 0;
+    virtual void *GetIClientRemotePlay(int, int) = 0;
     virtual unknown_ret __Destructor1() = 0;
 #if defined(ARGONX_UNIX)
     virtual unknown_ret __Destructor2() = 0;
diff --git a/IClientHTMLSurface.h b/IClientHTMLSurface.h
index b0ca7e9..b070cbe 100644
--- a/IClientHTMLSurface.h
+++ b/IClientHTMLSurface.h
@@ -1,7 +1,7 @@
 class IClientHTMLSurface
 {
 public:
-    virtual ~IClientHTMLSurface() = 0;
+    virtual unknown_ret __Destructor1() = 0;
 #ifdef ARGONX_UNIX
     virtual unknown_ret __Destructor2() = 0;
 #endif
